invenio_formatter/templates/semantic-ui/invenio_formatter/macros/badges.html

```diff
@@ -9,37 +9,44 @@
 
 {# Generate a list different ways to embed a badge. #}
 {%- macro badges_formats_list(image_filename, target, alternative='DOI') %}
+
+  <style>
```

Comment:
Minor: We should not add inline styles in the pages. I will need to check where to add it exactly, so no action needed on your side right now.

Reply:
I think this code has to go to Invenio-Theme on:

https://github.com/inveniosoftware/invenio-theme/blob/master/invenio_theme/assets/semantic-ui/less/invenio_theme/theme/collections/message.overrides